enhancements/service-catalog/prepare-service-catalog-and-brokers-for-removal.md

```diff
@@ -0,0 +1,173 @@
+--
+title: prepare-service-catalog-and-brokers-for-removal
+authors:
+  - "@jmrodri"
+reviewers:
+  - "@dmesser"
+  - "@joelanford"
+approvers:
+  - TBD
+creation-date: 2019-09-12
+last-updated: 2019-09-16
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+see-also:
+  - "/enhancements/this-other-neat-thing.md"
+replaces:
+  - "/enhancements/that-less-than-great-idea.md"
+superseded-by:
+  - "/enhancements/our-past-effort.md"
+---
+
+# prepare-service-catalog-and-brokers-for-removal
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+Operators became the strategy in OpenShift 4.x for customers to manage their
+applications in the cluster. As this technology has improved, the need for
+having a Service Catalog backed by Service Brokers is no longer needed. As an
+enterprise product, we need to deprecate the catalog and brokers, inform the
+users of this deprecation, and then finally remove them from the product in
+OpenShift 4.4. This should allow plenty of time to migrate any broker related
+items the customer may have had to Operators.
+
+## Motivation
+
+As our strategy shifted from Service Brokers to Operators, customers and
+partners need sufficient time to shift their strategy towards Operators as well.
+
+### Goals
+
+Service Catalog, Service Brokers and their associated operators need to be
+deprecated and removed by OpenShift 4.4.
+
+Alert users that the above have been deprecated and will be removed in a
+future OpenShift release.
+
+### Non-Goals
+
+Not in scope is writing the preflight-check for CVO.
```

Comment:
This is in scope for the enhancement, just not for 4.3 correct?

Reply:
What will the pre-flight checks do when svcat is removed in 4.4 then?

Reply:
Reworded it.